src/sonic-yang-models/yang-models/sonic-qos-pfc.yang

```diff
@@ -0,0 +1,82 @@
+module sonic-qos-pfc {
+    namespace "http://github.com/Azure/sonic-qos-pfc";
+    prefix sonic-qos-pfc;
+
+    yang-version 1.1;
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC QoS Priority-based Flow Control";
+
+    revision 2021-07-01 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-qos-pfc {
+
+        container PFC_WD {
+            list PFC_WD_LIST {
+                key "ifname";
+
+                leaf ifname {
+                    mandatory true;
+                    type string {
+                        pattern "Ethernet([1-3][0-9]{3}|[1-9][0-9]{2}|[1-9][0-9]|[0-9])|GLOBAL" {
+                            error-message "Invalid interface name";
```

Comment:
Looks like GLOBAL key is added only for polling interval, if that's the case, can we have the global configurations in the separate table since it looks like we're trying to overload both interface level & global level configurations in a single table.

Reply:
The Table name in CONFIG_DB is PFC_WD and it is overloaded to configure polling interval for GLOBAL key and watchdog parameters for an Ethernet interface. Adding a union to address this.

Reply:
Please have a seperate table for global configs.